### ISSUE TITLE
Need an output event

### ISSUE THREAD

**Issue by egodefroy** [webcat12345/ngx-intl-tel-input#374]
An event such as (change)="change($event)" with event the value of the phone number would be very useful to control mandatory fields, easily get the value of the field or other situations

rgds

**Comment by pasevin** [comment]
Yes that would be useful, I mentioned how it would be useful in this situation as well: https://github.com/webcat12345/ngx-intl-tel-input/issues/350

**Comment by egodefroy** [comment]
@pasevin could also be useful to easily set the phone value with [number]="value"
